lms/services/course.py

```diff
@@ -314,37 +262,6 @@ def is_member(self, course: Course, h_userid: str) -> bool:
             course.memberships.join(User).filter(User.h_userid == h_userid).first()
         )
 
-    def get_assignments(
```

Comment:
Remove this method, CourseService.get_assigmetns and replace it by calls to AssignmentService.get_assigmetns (with a course_id parameter).